views/data_table.tt

```diff
@@ -92,7 +92,9 @@
                 [% END %]
             </td>
             [% FOREACH column IN record.columns %]
-                <td class="[% column.type %]">[% render_datum(column) %]</td>
+                [% column_index = loop.index %]
+                [% col_header = columns.${column_index} %]
+                <td class="[% column.type %]">[% render_datum(column, col_header.name) %]</td>
```

Comment:
I think it would be better to add the column name to the ```column``` hash in ```GADS::Role::Presentation::Record```, then you could just do

    render_datum(column, column.name)

which I'm happy to do on merge, unless I've missed something?

Or even just do ```render_datum(column)``` and pull it out of the hash in ```render_datum```?